src/localplayer.cpp

```diff
@@ -329,12 +389,17 @@ void LocalPlayer::move(f32 dtime, Environment *env, f32 pos_max_d,
 		m_sneak_node = new_sneak_node;
 		m_sneak_node_exists = sneak_node_found;
 
-		// Update saved top bounding box of sneak node
 		if (sneak_node_found) {
+			// Update saved top bounding box of sneak node
 			MapNode n = map->getNodeNoEx(m_sneak_node);
 			std::vector<aabb3f> nodeboxes;
 			n.getCollisionBoxes(nodemgr, &nodeboxes);
 			m_sneak_node_bb_top = getTopBoundingBox(nodeboxes);
+
+			m_sneak_ladder_detected = physics_override_sneak_glitch &&
+					detectSneakLadder(map, nodemgr, floatToInt(position, BS));
```

Comment:
Unforunately this adds quite a lot complexity because I'd need to decouple the detection.
I doubt this is worth it performance-wise.